addons/binding/org.openhab.binding.nest/src/main/java/org/openhab/binding/nest/internal/data/NestMetadata.java

```diff
@@ -0,0 +1,18 @@
+package org.openhab.binding.nest.internal.data;
+
+import com.google.gson.annotations.SerializedName;
+
+public class NestMetadata {
```

Comment:
Please add a JavaDoc with `@author` tag containing a real name of the author. See [Code Style](http://docs.openhab.org/developers/development/guidelines.html#a-code-style).